src/librosco.mk

```diff
@@ -0,0 +1,20 @@
+# This file is part of MXE.
+# See index.html for further information.
+
+PKG             := librosco
+$(PKG)_IGNORE   :=
+$(PKG)_VERSION  := 0.1.7
+$(PKG)_CHECKSUM := e766ab05da2ce752f40df7f186674405506abe53252e91b3e21f1a87cefc593b
+$(PKG)_SUBDIR   := $(PKG)-$($(PKG)_VERSION)
+$(PKG)_FILE     := $(PKG)-$($(PKG)_VERSION).tar.gz
+$(PKG)_URL      := https://github.com/colinbourassa/librosco/archive/$($(PKG)_VERSION).tar.gz
+$(PKG)_DEPS     := gcc
+
+define $(PKG)_BUILD
+    mkdir '$(1)/build'
+    cd '$(1)/build' && cmake .. \
+        -DCMAKE_TOOLCHAIN_FILE='$(CMAKE_TOOLCHAIN_FILE)' \
+        -DBUILD_STATIC=$(if $(BUILD_STATIC),ON,OFF)
+
+    $(MAKE) -C '$(1)/build' -j '$(JOBS)' install
```

Comment:
Please separate building and installing to two commands. Build with `-j '$(JOBS)'`, install with `-j 1`.